sema4ai/vscode-client/src/robo/agentPackage.ts

```diff
@@ -287,8 +286,7 @@ export const refreshAgentSpec = async (agentPath: string): Promise<void> => {
         const errorMsg = `Failed to refresh the Agent Configuration`;
         logError(errorMsg, error, "ERR_REFRESH_AGENT_SPEC");
 
-        const detailedErrorMsg = `${errorMsg}. Please check the 'OUTPUT > Sema4.ai' for more details.`;
-        OUTPUT_CHANNEL.appendLine(detailedErrorMsg);
-        window.showErrorMessage(detailedErrorMsg);
+        OUTPUT_CHANNEL.appendLine(errorMsg);
```

Comment:
If logError(errorMsg) is used, the OUTPUT_CHANNEL.appendLine(errorMsg); is not needed (as logError will already add that to the OUTPUT_CHANNEL).